test/main-make-pr.ts

```diff
@@ -116,6 +118,20 @@ describe('Make PR main function', () => {
         expect(testMaintainersCanModify).equals(maintainersCanModify);
         expect(testPrimary).equals(primary);
       },
+      addLabels: (
```

Comment:
@bcoe Added stub helpers for what actually seems to be the main tests for `createPullRequest`.